tests/approval_tests/data/expected/reporter_catch2_xml_allfail

```diff
@@ -145,38 +145,41 @@
     <Failure filename="*testing_reporters.cpp" line="*">
       could not match caught std::runtime_error with expected content: could not find &apos;I throws&apos; in &apos;I threw&apos;
     </Failure>
-    <OverallResult success="false" durationInSeconds="*"/>
+    <OverallResult success="false" skips="0" durationInSeconds="*"/>
   </TestCase>
   <TestCase name="test unexpected throw fail" tags="" filename="*testing_reporters.cpp" line="*">
     <Failure filename="*testing_reporters.cpp" line="*">
       unexpected std::exception caught; message: unexpected error
     </Failure>
-    <OverallResult success="false" durationInSeconds="*"/>
+    <OverallResult success="false" skips="0" durationInSeconds="*"/>
   </TestCase>
   <TestCase name="test unexpected throw in section fail" tags="" filename="*testing_reporters.cpp" line="*">
     <Section name="section 1" filename="*testing_reporters.cpp" line="*">
       <Section name="section 2" filename="*testing_reporters.cpp" line="*">
-        <Failure filename="*testing_reporters.cpp" line="*">
-          unexpected std::exception caught; message: unexpected error
-        </Failure>
+        <OverallResults successes="0" failures="0" expectedFailures="0" skipped="true" durationInSeconds="*"/>
       </Section>
+      <OverallResults successes="0" failures="0" expectedFailures="0" skipped="true" durationInSeconds="*"/>
     </Section>
-    <OverallResult success="false" durationInSeconds="*"/>
+    <Failure filename="*testing_reporters.cpp" line="*">
+      unexpected std::exception caught; message: unexpected error
+    </Failure>
```

Comment:
I think this message is printed too late; I believe we want it to appear inside the `<Section>` where the exception was thrown, as it was before. See comment in `~section_entry_checker()`.

Reply:
Fixed.